binding/python/__init__.py

```diff
@@ -10,7 +10,18 @@
 #
 
 from .porcupine import Porcupine
-from .porcupine_error import *
+from .porcupine import PorcupineError
+from .porcupine import PorcupineMemoryError
+from .porcupine import PorcupineIOError
+from .porcupine import PorcupineInvalidArgumentError
+from .porcupine import PorcupineStopIterationError
+from .porcupine import PorcupineKeyError
+from .porcupine import PorcupineInvalidStateError
+from .porcupine import PorcupineRuntimeError
+from .porcupine import PorcupineActivationError
+from .porcupine import PorcupineActivationLimitError
+from .porcupine import PorcupineActivationThrottledError
+from .porcupine import PorcupineActivationRefusedError
```

Comment:
I guess there was a reason we did the error classes like this in the other packages: Because of how we package the bindings we can't have the unit tests and the package work at the same time. This can be resolved in the future if we want to split the files up like I tried here but the blast radius for that is a bit larger than the scope of this task.